engine/src/test/java/org/camunda/bpm/engine/impl/persistence/entity/util/TypedValueFieldTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.camunda.bpm.engine.impl.persistence.entity.util;
+
+import org.apache.commons.lang3.SerializationUtils;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.ProcessEngineConfiguration;
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.bpm.engine.impl.interceptor.CommandExecutor;
+import org.camunda.bpm.engine.impl.variable.serializer.JavaObjectSerializer;
+import org.camunda.bpm.engine.impl.variable.serializer.TypedValueSerializer;
+import org.camunda.bpm.engine.impl.variable.serializer.ValueFieldsImpl;
+import org.camunda.bpm.engine.impl.variable.serializer.VariableSerializerFactory;
+import org.camunda.bpm.engine.variable.Variables;
+import org.camunda.bpm.engine.variable.value.TypedValue;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TypedValueFieldTest {
+
+  private static final TypedValue INITIAL_VALUE = Variables.untypedValue(new Pojo("bar"));
+
+  private ProcessEngine engine;
+  private ProcessEngineConfigurationImpl configuration;
+
+  @Before
+  public void init() {
+    this.configuration = (ProcessEngineConfigurationImpl) ProcessEngineConfiguration.createStandaloneInMemProcessEngineConfiguration();
```

Comment:
You can use `ProcessEngineBootstrapRule` to create a new engine. For example, have a look at https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/test/java/org/camunda/bpm/engine/test/api/history/BulkHistoryDeleteDmnDisabledTest.java#L47-L49 how to use it. Also, you can declare it as a class rule that way, so that we avoid the costly operation of building a new engine as much as possible.

Reply:
Will do.